src/Sulu/Bundle/ContactBundle/Contact/CustomerManager.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Contact;
+
+use Doctrine\ORM\EntityManager;
+use Sulu\Bundle\ContactBundle\Util\IdsHandlingTrait;
+
+/**
+ * Implements functionality for the manager of account and contact combination.
+ */
+class CustomerManager implements CustomerManagerInterface
```

Comment:
i have created an issue for release 2.0 https://github.com/sulu-io/sulu/issues/1623